pkgs/os-specific/linux/rtl8852au/default.nix

```
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, kernel, bc, nukeReferences }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "rtl8852au";
-  version = "${kernel.version}-unstable-2024-04-16";
+  version = "${kernel.version}-unstable-2024-05-06";
 
   src = fetchFromGitHub {
     owner = "lwfinger";
-    repo = "rtl8852au";
-    rev = "5894bc6fed2bcaa525d13fcee1edada8aba67f2b";
-    hash = "sha256-R4Yb/jbh3nMgM41ByFjtkCMbsh/mmMRJ7CcvCRUvKu8=";
+    repo = pname;
```

```suggestion
    repo = "rtl8852au";
```

please undo this, this is considered an anti pattern

---

Done.